phpunit/class-wp-theme-json-test.php

```diff
@@ -315,6 +315,151 @@ function test_get_settings_appearance_false_does_not_opt_in() {
 		$this->assertEqualSetsWithIndex( $expected, $actual );
 	}
 
+	function test_get_settings_appearance_true_and_block_gap_false() {
```

Comment:
Do these tests reflect all the value types `blockGap` can have? This is: true, false, null, or a string.